lib/AutomaticEmails/WooCommerce/Events/FirstPurchase.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+namespace MailPoet\AutomaticEmails\WooCommerce\Events;
+
+use MailPoet\AutomaticEmails\WooCommerce\Helper as WCPremiumHelper;
```

Comment:
We should probably rename the namespace. But also, shall we move the content of `MailPoet\AutomaticEmails\WooCommerce\Helper`  into WooCommerce helper?

Reply:
Removed.